src/utilities/api/interactionManager.ts

```diff
@@ -105,3 +105,15 @@ export const makeSquarePayment = async (
       throw err;
     });
 };
+
+export const getVoucher = async (id: string) =>
+  axios
+    .get(vouchers + id)
+    .then((res) => res)
+    .catch((err) => err);
```

Comment:
Are errors caught/surfaced to the client?

Reply:
I think we can switch these to await calls instead of .then() and if there any errors they'll be thrown up the regular call stack instead of caught here--then maybe in the component catch the error and display a message?

Reply:
some of these api errors are caught and we either show the api error or make our own